app/src/views/private/components/sidebar-button.vue

```diff
@@ -6,41 +6,35 @@
 		@click="$emit('click', $event)"
```

Comment:
Unrelated to this PR, but this button will never work with `to` since it doesn't pass it on to the router-link

Reply:
Will be addressed separately 👍